bedrock/firefox/tests/test_redirects.py

```diff
@@ -5,13 +5,19 @@
 from unittest.mock import patch
 
 from django.conf import settings
-from django.test import RequestFactory, override_settings
+from django.test import RequestFactory
 
 import pytest
 
 from bedrock.firefox.redirects import mobile_app, validate_param_value
 
 
+# Custom decorator to skip Firefox.com redirect tests when ENABLE_FIREFOX_COM_REDIRECTS is False
+def skip_if_ffcom_redirects_disabled(func):
```

Comment:
```suggestion
def skip_if_fxc_redirects_disabled(func):
```

This is a total nit-pick but we've been abbreviating firefox.com as fxc in other places so it might be good to keep doing that here.

Reply:
Thanks for the context. I've update the decorator to be more abstract, so this isn't relevant anymore.